### PR TITLE
Kkmattil sd desktop software

### PR DESCRIPTION
**Pull request by kkmattil**
SD Desktop software list was reorganized and Apptainer instructions were updated.

### CODE REVIEW

File: docs/data/sensitive-data/sd-desktop-software.md [hunk 1]
```diff
@@ -1,52 +1,73 @@
...
+Upon logging in to the virtual desktop, you start a graphical Linux enviroment (Ubuntu22 or CentOS 7) with a standard set of software ready to use. This basic toolkit provides you a rather limited set of open-source software. This document lists of most commonly used default software tools in SD Desktop and represents methods that can be used to expand the software selection
 
-* SD Installer: A simple application for easy software additions.
-  
-* Apptainer and Podman: For those with technical expertise, create customized containers for specialized software setups.
+Please don't hesitate to contact [CSC Service Desk](../../support/contact.md) if you have any queistions about the software selection (subject: Sensitive Data). We can support you in your Desktop customization. Moreover, we are working on developing our services to provide proprietary software. Follow our webpage for constant updates.
```
Comment (ainoc) [inline docs/data/sensitive-data/sd-desktop-software.md:5]: queistions -> questions

**Review by kkmattil — COMMENTED** [review]
OK

**Comment by fmorelloCSC** [comment]
I asked ChatGTP to make a markdown-table with the software list, might be easier to search, I paste it here but you don't have to use it: 

| Category           | Software                                                                                                                                                                        |
|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Office tools       | LibreOffice                                                                                                                                                                     |
| Programming        | - go<br>- Python3 including packages: tensorflow, nltk, spacy, scikit-learn, seaborn, plotly, bokeh, pydot, xgboost, lightgbm, catboost, eli5, torch, keras, dist-keras, elephas, gensim, scrapy, beautifulsoup4, numpy, scipy, pandas, statsmodels, matplotlib, pyfuse3, crypt4gh, trio, httpx, qrcode |
|                    | - R / RStudio Server<br>  - Matching the r-env-singularity/4.0.5 module on Puhti (R 4.0.5 + RStudio Server 1.4.1106, Bioconductor 3.12)<br>  - 1000+ pre-installed R packages (versions available on Apr 20, 2021)<br>  - Intel® oneAPI Math Kernel Library (oneMKL)<br>  - GIS software (PROJ, GDAL, SAGA)<br>  - CUDA drivers<br>  - TensorFlow (used with R TensorFlow back-end)<br>  - R + RStudio Server versions matched with Puhti r-env-singularity module (R 4.0.5 + RStudio Server 1.4.1106, MRAN, Bioconductor 3.12) |
| Workflow engines   | - Nextflow<br>- Snakemake                                                                                                                                                       |
| Software containers | - Apptainer<br>- Podman (Only in Ubuntu machines)                                                                                                                                 |
| Scientific Software | - Plink 1.9<br>- Samtools 1.8<br>- Minimap2 2.26                                                                                                                                   |

**Comment by fmorelloCSC** [comment]
Should there be something after this sentence?
 However, the virtual desktop can also become a versatile research workspace designed to adapt to your specific needs **via customization with**:

for example?
- sd software installer (no previous knowledge needed)
- apatainers (advanced)

**Review by fmorelloCSC — APPROVED** [review]
